ntbs-service-unit-tests/Services/AlertServiceTest.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Moq;
+using ntbs_service.DataAccess;
+using ntbs_service.Models;
+using ntbs_service.Models.Enums;
+using ntbs_service.Services;
+using Xunit;
+
+namespace ntbs_service_unit_tests.Services
+{
+    public class AlertServiceTest
+    {
+        private readonly IAlertService mockAlertService;
+        private readonly Mock<IAlertRepository> mockAlertRepository;
+        private readonly Mock<INotificationRepository> mockNotificationRepository;
+        private readonly Mock<NtbsContext> mockContext;
+
+        public AlertServiceTest()
+        {
+            mockAlertRepository = new Mock<IAlertRepository>();
+            mockNotificationRepository = new Mock<INotificationRepository>();
+            mockContext = new Mock<NtbsContext>();
+            mockAlertService = new AlertService(mockAlertRepository.Object, mockNotificationRepository.Object);
+
+            mockContext.Setup(context => context.SetValues(It.IsAny<Object>(), It.IsAny<Object>()));
+        }
+
+        [Fact]
+        public async Task AddUniqueAlert_AddsIfNoAlertWithSameNotificationIdAndAlertType()
+        {
+            // Arrange
+            var y = new TestAlert() {AlertId = 2};
+            mockAlertRepository.Setup(x => x.GetAlertByNotificationIdAndTypeAsync(It.IsAny<int>(), It.IsAny<AlertType>()))

```

Comment:
you actually know what the id and types are going to be here - would supplying them be more readable?